tests/serverpod_test_server/lib/src/custom_classes.dart

```diff
@@ -31,15 +31,14 @@ class CustomClass2 {
   dynamic toJson() => {'text': value};
 }
 
-class CustomClass3 implements SerializableModel, ProtocolSerialization {
+class CustomClass3 implements SerializableModel {
```

Comment:
Suggestion: We should have a model field that uses ProtocolSerialization as well.

Add a test that calls this model to validate that the toJsonForProtcol value is retrieved.